.travis.yml

```diff
@@ -1,39 +1,34 @@
 language: scala
-
 dist: trusty
-
 sudo: false
-
 scala:
   - 2.11.11
```

Comment:
Looks like `2.11.12` is a thing?

Reply:
I'm happy to version bump that later but this has been languishing awhile, let's not hold it up.